ansible/roles/ami_9_aarch64/tasks/os.yaml

```diff
@@ -50,26 +50,22 @@
 - name: Install Base System on chroot
   ansible.builtin.shell:
     cmd: >
-      dnf -y --installroot=/rootfs --nogpgcheck --setopt=cachedir=/var/cache/dnf install
+      dnf -y --installroot=/rootfs --nogpgcheck --setopt=install_weak_deps=False --setopt=cachedir=/var/cache/dnf install
```

Comment:
What's the reason behind disabling gpgcheck here? Does it fail in the chroot?

Reply:
Good question.

This options prevents `dnf` to add a new `gpg-pubkey*` entry to the RPM database for the `RPM-GPG-KEY-AlmaLinux-(8|9|10)`. Which shows the prompts below on the example below:

```sh
Total                                                                                                                            78 MB/s | 102 MB     00:01     
AlmaLinux 9 - BaseOS                                                                                                            2.5 MB/s | 3.1 kB     00:00    
Importing GPG key 0xB86B3716:
 Userid     : "AlmaLinux OS 9 <packager@almalinux.org>"
 Fingerprint: BF18 AC28 7617 8908 D6E7 1267 D36C B86C B86B 3716
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux-9
Is this ok [y/N]:
```

So, It is just sake of improving the user experience (like using a freshly installed AlmaLinux OS system).

Reply:
That makes sense! Thanks for taking the time to explain it :).